include/pcl_conversions/pcl_conversions.h

```diff
@@ -77,27 +76,28 @@ namespace pcl_conversions {
   /** PCLHeader <=> Header **/
 
   inline
-  void fromPCL(const pcl::uint64_t &pcl_stamp, ros::Time &stamp)
+  void fromPCL(const pcl::uint64_t &pcl_stamp, builtin_interfaces::msg::Time &stamp)
   {
-    stamp.fromNSec(pcl_stamp * 1000ull);  // Convert from us to ns
+    stamp.sec = (int32_t)(pcl_stamp / 1000000ull); // Convert whole seconds from us to sec
+    stamp.nanosec = (uint32_t)((pcl_stamp % 1000000ull) * 1000ull); // Convert remainder to nanosec
   }
 
   inline
-  void toPCL(const ros::Time &stamp, pcl::uint64_t &pcl_stamp)
+  void toPCL(const builtin_interfaces::msg::Time &stamp, pcl::uint64_t &pcl_stamp)
   {
-    pcl_stamp = stamp.toNSec() / 1000ull;  // Convert from ns to us
+    pcl_stamp = stamp.sec * 1000000ull + stamp.nanosec / 1000ull;  // Convert to us
```

Comment:
So, I changed it because technically we aren't converting from nanoseconds anymore; we are converting from sec/nsec together.  It's totally clear what we are converting *from* based on the code; I thought the comment might be helpful for what we are converting *to*.  However, I wonder if it might just be better to change "pcl_stamp" to "pcl_stamp_us", and just get rid of the comment completely.  What do you think?